hello-world/server_test.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
it would be good to add add a call to run these tests in the workflow file.

Reply:
I've added the github action to test all images with Go source code.